libs/generators/src/lib/generators/generators-menu/generators-menu.component.ts

```diff
@@ -13,5 +14,10 @@ import { MatSidenavModule } from '@angular/material/sidenav';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class GeneratorsMenuComponent {
-  public menuItems = ['Component', 'Directive', 'Module', 'Pipe'];
+  public readonly router = inject(Router);
+  public menuItems = ['component', 'directive', 'module', 'pipe'];
```

Comment:
It would be cool if we won't need a separate list for this and instead we'll reuse the routes config as the sidenav items

Reply:
@ronnetzer see prev comment.

Reply:
@eladzipper96 I agree with @ronnetzer and the benefit of that is that your code is not decoupled, display value and URL should not be dependent on each other